src/main/java/com/google/api/generator/engine/ast/OperatorKind.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.ast;
+
+import java.util.EnumSet;
+
+public enum OperatorKind {
+  ARITHMETIC_ADDITION,
+  RELATIONAL_EQUAL_TO,
+  RELATIONAL_NOT_EQUAL_TO,
+  UNARY_LOGICAL_NOT,
+  UNARY_POST_INCREMENT;
+
+  private static final EnumSet<OperatorKind> PREFIX_OPERATORS_SET;
+
+  static {
```

Comment:
Have you tried just assigning this outside of the static block?

Reply:
Assigned outside of static block.

Previously, It showed the compiler error  "constructor can't access static field" if I declare `private EnumSet<OperatorKind> PREFIX_OPERATORS_SET = EnumSet.of(..)`. So I put static block. 

But Turns out if I declare it as static variable will fix that compiler error.